### ISSUE TITLE
Listen for elapsed time, not just calls

### ISSUE THREAD

**Issue by hauleth** [Stratus3D/eflambe#48]
When profiling some functions, especially infinitely looping functions, it is easier to trace them for given period instead of listening for given amount of function calls. Would that be possible to implement?

**Comment by Stratus3D** [comment]
Yes, I think it'd be possible. I don't have time for feature work on this right now. Focusing on asdf for the next couple months.
